ams/server/src/main/java/com/netease/arctic/server/dashboard/PaimonTableDescriptor.java

```diff
@@ -169,23 +174,32 @@ public ServerTableMeta getTableDetail(AmoroTable<?> amoroTable) {
   }
 
   @Override
-  public List<AmoroSnapshotsOfTable> getSnapshots(AmoroTable<?> amoroTable, String ref) {
-    if (ref != null) {
-      throw new UnsupportedOperationException("Paimon not support tag and branch");
-    }
+  public List<AmoroSnapshotsOfTable> getSnapshots(
+      AmoroTable<?> amoroTable, String ref, OperationType operationType) {
     FileStoreTable table = getTable(amoroTable);
     List<AmoroSnapshotsOfTable> snapshotsOfTables = new ArrayList<>();
     Iterator<Snapshot> snapshots;
-    try {
-      snapshots = table.snapshotManager().snapshots();
-    } catch (IOException e) {
-      throw new RuntimeException(e);
+    if ("main".equals(ref)) {
```

Comment:
I think magic words could be avoided.

Reply:
Yes, maybe define a static value for that.

Reply:
sure